### ISSUE TITLE
Latest versions continue to crash

### ISSUE THREAD

**Issue by sabunassar** [AdguardTeam/AdguardBrowserExtension#2768]
### AdGuard Extension version

4.3.35

### Browser version

Chome 123

### OS version

Windows 10

### What filters do you have enabled?

AdGuard Base filter

### What Stealth Mode options do you have enabled?

Block trackers

### Issue Details

Open Chrome & open any web page. Within 10 seconds, Chrome reports AdBlock has crashed & allows to reload extension.  Reloading is ok for a few seconds and then it crashes again.  The loop continues.  I have enabled error collection and am providing a screenshot of the error.

### Screenshots

<details><summary>Error Collection Screenshot:</summary>

![image](https://github.com/AdguardTeam/AdguardBrowserExtension/assets/129466193/d79d9a96-73ba-41c3-9fc5-e0a65978b1d3)

</details>

**Comment by alexx7311** [comment]
@sabunassar Hello! Please do the following: 
1. Quit any running instance of Chrome.
2. Right-click your Chrome shortcut.
3. Select **Properties**.
4. At the end of the **Target**: line, add the command line flags: `--enable-logging --v=1`
5. With that example flag, it should look like: `chrome.exe --enable-logging --v=1`
6. Click `Apply` and then `OK`.
7. Launch Chrome using your shortcut.
8. Reproduce the problem.

The file with debug logs will be stored in the user data directory as chrome_debug.log
Please attach the file to the comment in this issue.

**Comment by sabunassar** [comment]
Requested log file uploaded.
[chrome_debug.log](https://github.com/AdguardTeam/AdguardBrowserExtension/files/14820149/chrome_debug.log)

**Comment by maximtop** [comment]
@sabunassar Have you collected the logs during the crash and repair? I wasn't able to find anything related to this.

**Comment by alexx7311** [comment]
@sabunassar The log will rewrite itself every time you reopen the browser. You can reproduce the problem, close the browser, move the log to the desktop for example, and then open Chrome again if you need to.

**Comment by sabunassar** [comment]
Yes, I did copy out the file prior to re-starting Chrome. I also created a separate icon to launch debug version.  I did it again & captured the log.  AdGuard consistently crashes after ~27 seconds & Chrome is completely idle (I did not open any pages). I have no other extensions installed & the launch page is just the empty New TAB page.  I should also note this is Windows 8.1 so Chrome is last supported version (109.0.5414.129). I don't see the crash on newer Chrome (123.x) in Windows 10 but have never had an issue on the older version until this past week.

I just removed the extension & re-installed without modifying any of the settings. Will report if that resolved the issue.

[chrome_debug.log](https://github.com/AdguardTeam/AdguardBrowserExtension/files/14826980/chrome_debug.log)
![AdGuard_crash](https://github.com/AdguardTeam/AdguardBrowserExtension/assets/129466193/e5f620cc-369f-43f1-9fe0-daf60cb0befd)
